keyboards/melody96/hotswap/hotswap.h

```diff
@@ -0,0 +1,108 @@
+	{ K100,  K101,  K102,  KC_NO, K104,  K105,  K106,  K107,  K108  }, \
+	{ K110,  K111,  K112,  K113,  K114,  K115,  K116,  K117,  K118  }  \
+}
+
+/*
+ * The layout macro for 96_ansi (YMDK layout C) with illustrative grid of a typical assignment.
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+ * │Esc│F1 │F2 │F3 │F4 │F5 │F6 │F7 │F8 │F9 │F10│F11│F12│PSc│Scr│Pse│Vo-│Vo+│Mut│
+ * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┴───┼───┼───┼───┼───┤
+ * │ ` │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ Backsp│Num│ / │ * │ - │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┼───┼───┼───┤
+ * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │ 7 │ 8 │ 9 │   │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┼───┼───┤ + │
+ * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │  Enter │ 4 │ 5 │ 6 │   │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┼───┼───┼───┤
+ * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │Shift │ ↑ │ 1 │ 2 │ 3 │   │
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬┴──┬───┼───┼───┼───┼───┤Ent│
+ * │Ctrl│GUI │Alt │         Space          │Alt│GUI│Ctl│ ← │ ↓ │ → │ 0 │ . │   │
+ * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
+ */
+#define LAYOUT_96_ansi( \
+  K050, K051, K052, K053, K054, K055, K056, K057, K058, K118, K117, K115, K114, K113, K116, K112, K111, K110, K063, \
+  K040, K041, K042, K043, K044, K045, K046, K047, K048, K108, K107, K105, K104, /***/ K106, K102, K101, K100, K064, \
+  K030, K031, K032, K033, K034, K035, K036, K037, K038, K098, K097, K095, K094,       K093, K096, K092, K091, /***/ \
+  K020, K021, K022, K023, K024, K025, K026, K027, K028, K088, K087, K085,             K084, K086, K082, K081, K080, \
+  K010, /***/ K012, K013, K014, K015, K016, K017, K018, K078, K077, K075,       K074, K073, K076, K072, K071, /***/ \
+  K000, K001, K002,                   K006,                   K008, K007, K005, K004, K003, K066, K062, K061, K060  \
+) { \
+	{ K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008  }, \
+	{ K010,  KC_NO, K012,  K013,  K014,  K015,  K016,  K017,  K018  }, \
+	{ K020,  K021,  K022,  K023,  K024,  K025,  K026,  K027,  K028  }, \
+	{ K030,  K031,  K032,  K033,  K034,  K035,  K036,  K037,  K038  }, \
+	{ K040,  K041,  K042,  K043,  K044,  K045,  K046,  K047,  K048  }, \
+	{ K050,  K051,  K052,  K053,  K054,  K055,  K056,  K057,  K058  }, \
+	{ K060,  K061,  K062,  K063,  K064,  KC_NO, K066,  KC_NO, KC_NO }, \
+	{ KC_NO, K071,  K072,  K073,  K074,  K075,  K076,  K077,  K078  }, \
+	{ K080,  K081,  K082,  KC_NO, K084,  K085,  K086,  K087,  K088  }, \
+	{ KC_NO, K091,  K092,  K093,  K094,  K095,  K096,  K097,  K098  }, \
+	{ K100,  K101,  K102,  KC_NO, K104,  K105,  K106,  K107,  K108  }, \
+	{ K110,  K111,  K112,  K113,  K114,  K115,  K116,  K117,  K118  }  \
+}
+
+#define LAYOUT LAYOUT_all
+#define LAYOUT_c LAYOUT_96_ansi
```

Comment:
```suggestion
```

Reply:
Thanks for the feedback -- I'll add the license headers. What's the rationale for removing the LAYOUT aliases? I was following the style of [whitefox](https://github.com/qmk/qmk_firmware/blob/master/keyboards/whitefox/whitefox.h) based on fauxpark's suggestions.

This is still a work in progress, as I need a little bit of guidance as far as the compilation errors I'm having with the hotswap layout. Not sure of the folder structure yet.